Neos.Flow/Documentation/TheDefinitiveGuide/PartIII/Logging.rst

```diff
@@ -107,16 +107,18 @@ You might already have stumbled across such an exception (though hopefully not!)
   HTTP REQUEST:
   127.0.0.1:8081keep-aliveno-cacheno-cacheimageMozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.132 Safari/537.361image/webp,image/apng,image/*,*/*;q=0.8same-originno-corshttp://127.0.0.1:8081/flow/welcomegzip, deflate, brde-DE,de;q=0.9,en-US;q=0.8,en;q=0.7
 
-  HTTP RESPONSE:
```

Comment:
The response is no longer logged as of Flow 7 (and was totally useless before anyway, as it was just a dummy response).